website/vercel.json

```diff
@@ -0,0 +1,13 @@
+{
+  "headers": [
+    {
+      "source": "/(.*)",
```

Comment:
I imagine there is a way to disable the indexing of certain pages. In that case, would it be possible to disable the indexing of the "testing" section? https://hds-website-hashicorp.vercel.app/testing

Reply:
Yes, good call – I'll set that up